handsontable/src/plugins/formulas/__tests__/formulas.spec.js

```diff
@@ -2951,6 +2951,65 @@ describe('Formulas general', () => {
     ]);
   });
 
+  describe('renaming sheet for HF instance', () => {
+    it('should update HOT\'s plugin internal property', () => {
+      let hookTriggered = false;
+      const hfInstance = HyperFormula.buildEmpty({});
+      const hot = handsontable({
+        data: [
+          ['01/03/1900'],
+          ['=A1']
+        ],
+        formulas: {
+          engine: hfInstance,
+          sheetName: 'Sheet1'
+        },
+        columns: [{
+          type: 'date',
+          dateFormat: 'DD/MM/YYYY'
+        }],
+      });
+
+      hot.addHook('afterSheetRenamed', () => {
+        expect(hot.getPlugin('formulas').sheetName).toBe('Lorem Ipsum');
```

Comment:
I think that it's a bad idea to make expectations in the hook callbacks. When the hook somehow stops working (no callback call), there won't be a test error or warning or so.

I'd suggest something like that
```js
let sheetNameViaHook;

hot.addHook('afterSheetRenamed', () => {
  sheetNameViaHook = hot.getPlugin('formulas').sheetName;
});

expect(sheetNameViaHook).toBe('Lorem Ipsum');
```

Reply:
That's why I've added an extra variable: https://github.com/handsontable/handsontable/pull/10719/files#diff-d5c22bed64b93701cdf3ae18b046941d8898c7d8826d7d032e5760437d78b687R2976 :) Anyway, I can change it to make oneliner.